Makefile

```diff
@@ -19,14 +19,30 @@ ci:
 ci-down:
 	$(COMPOSE_TEST) down
 
-.PHONY: ci-spec
-ci-spec:
-	@$(BASH_TEST) "bin/rspec ${SPEC_PATH}"
-
 .PHONY: bash
 bash:
 	@$(COMPOSE_DEV) $(BASH)
 
+.PHONEY: ci-build
```

Comment:
```suggestion
.PHONY: ci-build
```

Reply:
i see the same spelling (PHONEY) for `ci-down` too --or are they supposed to be like that?

Reply:
whoops, nope. good eye 👍

Reply:
That is most certainly my fault, thank you for catching it!